sample_data/project-list.md

```diff
@@ -27,13 +27,15 @@ open:
 - <https://gitlab.com/librespacefoundation>
   - An open source hardware satellite
 - <https://github.com/orbitalindex/awesome-space#spacecraft-hardware>
+  - List of spacecraft hardware (including ground equipment), some of which are open source hardware
 - <https://github.com/Opentrons/ot2>  (ot2 already in the list)
   - Opentrons is a highly commercially successful biology lab automation platform
 - precious plastics machines
 - wikihouse
 - <https://www.latelierpaysan.org/Technical-drawings-tutorials>
 - <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4853539/> (medical devices)
 - <https://www.open-raman.org/>
+  - Unfortunately, OpenRAMAN is *not* open source hardware because its designs are released under [CC BY-NC 4.0](https://creativecommons.org/licenses/by-nc/4.0/) which is not an open source license.
```

Comment:
Could you point out where you found this information?

Just downloaded the design files from [here](https://www.open-raman.org/build/starter-edition/download/); they use CERN OHL-w (→ v2)

However, the original design files are missing, but as they provide STEP files _and_ PDF drawings, I'm pretty sure they still have them.

In general, I'd be in favour moving projects with misalignments into the bottom section of the document "to add after misalignments are clarified:" followed by a specification of the misalignment.

Reply:
Oooooh. Wow!!

I think they've updated the license to CERN OHL-w 2.0 *after* I last visited their site. That's great news! Let's remove this line, the.